src/webserver.cpp

```diff
@@ -463,7 +463,7 @@ MHD_Result webserver::post_iterator(void *cls, enum MHD_ValueKind kind,
 
     try {
         if (filename == nullptr || mr->ws->file_upload_target != FILE_UPLOAD_DISK_ONLY) {
-            mr->dhr->set_arg(key, mr->dhr->get_arg(key) + std::string(data, size));
+            mr->dhr->set_arg(key, std::string(mr->dhr->get_arg(key)) + std::string(data, size));
```

Comment:
Why do we need the explicit cast here? Doesn't string_view have a conversion constructor that does it transparently?  (asking for the various places where this happens, not only this instance)

Reply:
Yes, and nope: https://stackoverflow.com/questions/47525238/why-is-there-no-implicit-conversion-from-stdstring-view-to-stdstring

Reply:
makes sense